org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ChangesImpl.java

```diff
@@ -101,15 +105,30 @@ public void log(Logger logger, String inputPath, String outputPath) {
 				logger.info(consoleMarker, "Input  [ {} ] as [ {} ]", getInputResourceName(), inputPath);
 			}
 			if (useOutputName.equals(outputPath)) {
-				logger.info(consoleMarker, "Output [ {} ]", outputPath);
+				logger.info(consoleMarker, "Output [ {} ] took [ {} ] ms", outputPath,
```

Comment:
You format the value as HHMMSS but put "ms" after the value.

Reply:
Yes, good catch, it used to be ms, but for WAR where it takes 699543ms, it's not human readable